### DISCUSSION TITLE
Jenny dataset

### DISCUSSION THREAD

**Discussion by hobodrifterdavid** [coqui-ai/TTS#2266] [General]
Hi. We made an English voice dataset last year. It can be used for any purpose with only attribution. The clips need need trimming at least, I didn't get around to it. Perhaps you find a use for it. :) Further info here:

https://github.com/dioco-group/jenny-tts-dataset

**Comment by Pantyhose-X** [comment]
Request a clone of [YouTube Lucy](https://www.youtube.com/@EnglishwithLucy) voice

**Comment by erogol** [comment]
> Hi. We made an English voice dataset last year. It can be used for any purpose with only attribution. The clips need need trimming at least, I didn't get around to it. Perhaps you find a use for it. :) Further info here:
> 
> https://github.com/dioco-group/jenny-tts-dataset

thanks for sharing... If you want you can send a PR for a Jenny recipe.
